sisl/viz/backends/plotly/backend.py

```diff
@@ -430,11 +434,13 @@ def draw(self, drawer_info, rows, cols, childs, **make_subplots_kwargs):
 
 class PlotlyAnimationBackend(PlotlyBackend, AnimationBackend):
 
-    def draw(self, drawer_info, childs, get_frame_names):
-        frames_layout = self._build_frames(childs, None, get_frame_names)
+    def draw(self, backend_info):
+        childs = backend_info["child_plots"]
+        frame_names = backend_info["frame_names"]
+        frames_layout = self._build_frames(childs, None, frame_names)
         self.update_layout(**frames_layout)
 
-    def _build_frames(self, childs, ani_method, get_frame_names):
+    def _build_frames(self, childs, ani_method, frame_names):
```

Comment:
I think the common name is children, not childs

Reply:
True